test/data/users.js

```diff
@@ -51,3 +51,8 @@ exports.pagination = {
   page: 1,
   limit: 1
 };
+
+exports.token = {
+  token:
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
```

Comment:
Doesn't this token expire?

Reply:
Yes it does, in fact, it's a wrong token that helps me test that case in particular

Reply:
Got it